### PR TITLE
floating-navbar

### PR DESCRIPTION
**Pull request by Mbrick2**
Made by @BarelySmooth

### CODE REVIEW

File: addons/addons.json [hunk 1]
```diff
@@ -116,7 +116,7 @@
   "search-sprites",
   "hide-signatures",
   "disable-sprite-wobble",
-
+  "floating-navbar"
```
Comment (GrahamSH-LLK) [inline addons/addons.json:119]: uhh this should have a comma after it 
also add the line break back

Comment (Mbrick2) [reply addons/addons.json]: Fixed

Comment (Mbrick2) [reply addons/addons.json]: [Resolved](https://github.com/ScratchAddons/ScratchAddons/pull/4155/commits/f323eed81d55f060fd3e7cccb87e5b0fd95f3260)

**Comment by kccuber-scratch** [comment]
apparently, firefox doesn't support backdrop-filter with blur. I believe there's some hacky ways to get this to work but currently it's not supported.

**Comment by apple502j** [comment]
Marked as draft until we have proper description, checking whether we can drop firefox, etc

**Comment by Secret-chest** [comment]
what does this do?

**Comment by lisa-wolfgang** [comment]
I was told by a Nightly tester that this doesn't work on the Home or Explore pages.

**Comment by Mbrick2** [comment]
> Marked as draft until we have proper description, checking whether we can drop firefox, etc

@apple502j added

**Comment by Mbrick2** [comment]
@WorldLanguages can this get merged

**Comment by Dhuls-Scratch** [comment]
> @WorldLanguages can this get merged

read all of the below


> I was told by a Nightly tester that this doesn't work on the Home or Explore pages.



> apparently, firefox doesn't support backdrop-filter with blur. I believe there's some hacky ways to get this to work but currently it's not supported.
> 
> EDIT: Oh yeah I forgot that the Firefox developers are working on it.



> Marked as draft until ~~we have proper description~~ (resolved by [255990c](https://github.com/ScratchAddons/ScratchAddons/pull/4155/commits/255990cfa53a1788f006361a058d9dc23c8d4e64)), checking whether we can drop firefox, etc

**Comment by Secret-chest** [comment]
how does it look?

**Comment by Weredime** [comment]
@Mbrick2 Please enable actions on your fork

**Comment by GrahamSH-LLK** [comment]
Is this still under development?

**Comment by WorldLanguages** [comment]
@mxmou Do you think this is worth adding? This is how it looks:

brave_MvLymbPaeO

**Comment by WorldLanguages** [comment]
This would be potentially incompatible with this recent change: #5788

**Comment by BroJac5246** [comment]
> This would be potentially incompatible with this recent change: #5788

If we add this then it should be part of that addon.

**Comment by mxmou** [comment]
> @mxmou Do you think this is worth adding?

It looks nice but I don't think it fits with Scratch's design.

**Comment by WorldLanguages** [comment]
Okay. I'm closing this then.
It can be reopened if we continue to discuss the addon idea in a "new addon" issue and conclude it's worth adding.

**Pull request closed without merging**